meinberlin/apps/contrib/templates/meinberlin_contrib/item_detail.html

```diff
@@ -127,6 +127,60 @@ <h2 class="moderator-statement__title">{% trans 'Official Statement' %}</h2>
 
     {% block vote_button %}{% endblock %}
 
+    {% block moderation_info %}
+        {% if is_moderator and module.blueprint_type in 'PB3,PB2,PB1' %}
+        <div class="offset-lg-3 col-lg-6">
+            <div class="accordion">
+                <div class="moderation-info__accordion-title">
+                    <h2 id="accordion-moderation-title" class="moderation-info__title">{% translate 'Moderation' %}</h2>
+
+                    <a
+                        href="#accordion-moderation-body"
+                        class="collapsed"
+                        aria-haspopup="true"
+                        aria-expanded="true"
+                        data-bs-toggle="collapse">
+                            <i class="fa fa-chevron-down moderation-info__accordion-chevron" aria-hidden="true"></i>
+                    </a>
+                    {% get_item_url object 'moderate' False as moderate_url %}
+                    {% if moderate_url %}
+                        <a
+                            href="{{ moderate_url }}"
+                            class="btn btn--light btn--small moderation-info__btn"
+                            data-embed-target="external">
+                                <i class="fas fa-pencil" aria-hidden="true"></i>
+                                {% translate 'Edit' %}
+                        </a>
```

Comment:
i'm not sure on the a11y of a button within a link as it doesn't make much sense to a sr, as the expand link is now not connected to the title, I think this may need a think and maybe a Marcelo chat